RedHatInsights/drift-frontend
# fix: DRFT-1208 - Pass app store to components in FEC requiring a store

This PR changes how the redux store for Drift is created, to be able and pass it properly and ensure that it is always the same instance. Without this visiting Drift could under certain conditions fail to get a proper store in dependent Components like `Main` and `NotificationsPortal` coming from `frontend-components`.

How to test:
1) Visit the "Subscriptions" application and reload to ensure that assets will come from that app.
2) Visit the "Drift" application.
3) With this fix the application should load and work properly. (Without it an error would be raised)

**Note:** This is a temporary patch and will hopefully be revertible at some point.

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
